includes/documents/class-wcpdf-sequential-number-store.php

```diff
@@ -110,18 +110,23 @@ public function increment( $order_id = 0, $date = null ) {
 	 * Get the number that will be used on the next increment
 	 * @return int next number
 	 */
-	public function get_next() {
+	public static function get_next( $store_name, $method ) {
```

Comment:
if you can avoid making this static, that has my preference for better backwards compatibility (same for the usage of function arguments, which 3rd party integrations do not have!)